### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by KX5-jeron** [Lailloken/Exile-UI#427] [Q&A]
Is there an easy way to update?? going to try the new POE season and wanted to use your program. should I just delete the old folder and use the new one I downloaded???

**Comment by Lailloken** [comment]
Depends on when you used it last. If you see an `updater` section in the settings menu, you can use that. If it doesn't exist, it's probably best to do a clean install and start from scratch.

**Reply by KX5-jeron** [reply]
I used it last season.. I did a clean install because I wasn't sure.. but after seeing your comment I found the updater in settings and will use that from now on.. Thanks, Love your program btw..
